src/components/Profile.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import RenderJoined from './renderJoined';
+
+const Profile = () => {
+  const reservedMission = useSelector((store) => store.mission.missions)
+    .filter((mission) => mission.reserved);
+  const reservedRockets = useSelector((store) => store.rockets.rockets)
+    .filter((rocket) => rocket.reserved);
+  const reservedDragons = useSelector((store) => store.dragon.dragons)
+    .filter((dragon) => dragon.reserved);
+  return (
+    <ul className="reserved-sections">
+      <li className="section">
+        <RenderJoined list={reservedMission} render="missions" />
+      </li>
+      <li className="section">
+        <RenderJoined list={reservedRockets} render="rockets" />
+      </li>
+      <li className="section">
+        <RenderJoined list={reservedDragons} render="dragons" />
+      </li>
```

Comment:
[ OPTIONAL CHANGES ]

- [ ] Just as the previous reviewer recommended, it would be nice if you could add a conditional display so that when a rocket, dragon, or mission is empty, it shows a text like - No reserved rocket, and when there is one, then the rocket shows up. The same can be done for the rest. This will add nice user-friendliness into your app 👍🏽